v2/pkg/apis/kubeflow/v2beta1/types.go

```diff
@@ -44,9 +44,8 @@ type MPIJobSpec struct {
 	// +optional
 	SlotsPerWorker *int32 `json:"slotsPerWorker,omitempty"`
 
-	// CleanPodPolicy defines the policy that whether to kill pods after the job completes.
-	// Defaults to None.
-	CleanPodPolicy *common.CleanPodPolicy `json:"cleanPodPolicy,omitempty"`
+	// RunPolicy encapsulates various runtime policies of the job.
+	RunPolicy common.RunPolicy `json:"runPolicy"`
```

Comment:
Great. we will have unified interface now. All rest operator have been change to similar pattern.